service/server.go

```diff
@@ -1,17 +1,20 @@
 package main
 
 import (
+	"encoding/json"
 	"flag"
-	"net/http"
-	"strings"
-
+	"fmt"
 	log "github.com/Sirupsen/logrus"
 	"github.com/gorilla/mux"
-	lru "github.com/hashicorp/golang-lru"
+	"github.com/hashicorp/golang-lru"
```

Comment:
Good to know the "lru" removed in front, the import works. Last time when I did not have "lru" in front of the import path, compiler complains.